docs/business.yaml

```diff
@@ -3,8 +3,33 @@ info:
   title: businesses
   version: '2.0'
   description: |-
-    Businesses provideas all of the services needed to manage legal entities.
-    This encompasses all types of businesses authorized to do business by the Registrar.
+    The Business Registry is a centralized electronic register of all business in British Columbia, This encompasses all types of businesses authorized to do business by the Registrar. The Business API provides a secure means of integrating your business systems with the Registry.
```

Comment:
The Business Registry is a centralized electronic register of all business in British Columbia, (Maybe it is better to change the ',' to '.' here)This encompasses all types of businesses authorized to do business by the Registrar.